Library/Homebrew/utils/analytics.sh

```diff
@@ -0,0 +1,71 @@
+setup-analytics() {
+  [[ -z "$HOMEBREW_AUTO_UPDATE" ]] && return
+  [[ -n "$HOMEBREW_NO_AUTO_UPDATE" ]] && return
```

Comment:
My code examples shouldn't always be copied verbatim. :wink: Good thing it was noticed before it could create any confusion. :bow: